src/platform/lumin-runtime/elements/builders/color-picker-builder.js

```diff
@@ -1,16 +1,19 @@
 // Copyright (c) 2019 Magic Leap, Inc. All Rights Reserved
 
-import { ui, VEC4_ONE } from 'lumin';
+import { ui } from 'lumin';
 
 import { UiNodeBuilder } from './ui-node-builder.js';
-import { ArrayProperty } from '../properties/array-property.js';
+import { ColorProperty } from '../properties/color-property.js';
+
 import { PropertyDescriptor } from '../properties/property-descriptor.js';
 
+const VEC4_ONE = [255, 255, 255, 1];
```

Comment:
I understand that you did this to keep the change small but please pick a better name and especially one that won't conflict.

Reply:
👍